servo/checks.py

```diff
@@ -735,6 +737,152 @@ class Config:
         extra = pydantic.Extra.allow
 
 
+class CheckHelpers(pydantic.BaseModel, servo.logging.Mixin):
+    @classmethod
+    async def process_checks(
+        cls,
+        checks_config: servo.configuration.ChecksConfiguration,
+        results: list,
+        passing: set,
+    ) -> bool:
+
+        remedy = checks_config.remedy
+        check_halting = checks_config.check_halting
```

Comment:
These variables are used once each, please reference the config directly for brevity and clarity on lines 791 and 799